cpp/include/cudf/quantiles.hpp

```diff
@@ -94,5 +94,29 @@ std::unique_ptr<table> quantiles(
   std::vector<null_order> const& null_precedence = {},
   rmm::mr::device_memory_resource* mr            = rmm::mr::get_current_device_resource());
 
+/**
+ * @brief Calculate approximate percentiles on an input tdigest column.
+ *
+ * tdigest (https://arxiv.org/pdf/1902.04023.pdf) columns are produced specifically
+ * by the TDIGEST and MERGE_TDIGEST groupby aggregations.  These columns represent
```

Comment:
This shouldn't need to be groupby only, right? You can produce a tdigest for a single, entire column and wish to compute the approx percentile from that? 
```suggestion
 * by the TDIGEST and MERGE_TDIGEST aggregations.  These columns represent
```

Reply:
You certainly could, but internally the whole thing is currently implemented in terms of groupby in the sense that:
- It requires sorted inputs (for the non merge case). 
- It requires the groupby.group_labels() and group offsets/sizes.

Reply:
Sure, but tidget is only supported in groupby currently, but there's nothing to say it couldn't eventually be supported in `cudf::reduce`, yes?

Reply:
That's true.